server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/MinorCompactionTask.java

```diff
@@ -111,7 +111,7 @@ public void run() {
       if (tablet.needsSplit()) {
         tablet.getTabletServer().executeSplit(tablet);
       }
-    } catch (Throwable t) {
+    } catch (Exception t) {
```

Comment:
This seems to be wrapping the exception with a RTE. Are there even checked exceptions that can occur here? If so, we probably should enumerate them explicitly, and only wrap checked exceptions, leaving RTEs unwrapped.